### PR TITLE
State page layout

### PR DESCRIPTION
**Pull request by landonreed**
This PR starts to address #74 (state page layout). It:
- moves shared modals to common components
- expands the container to be full width (`.container-fluid`) to match mocks
- improves the layout a bit (placeholder image, modals, placeholder social buttons)

Note: there are a few weird styling issues with the open modal buttons, but I wasn't sure the best way to address the styling at this stage.

## Testing
1. Since this is a forked repo, to check out locally you can use the [GH CLI](https://github.com/cli/cli) to checkout locally (thanks for the tip @DanielKim1):
```
gh pr checkout 89
```

2. To view state page changes: go to http://localhost:3000/georgia

image

### CODE REVIEW

File: components/common/ReportMissingInfo.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+const ReportMissingInfo = () => (
+    <Modal target="report-missing-info"
+        text="Report missing or outdated information"
+        title="Report missing or outdated information">
+        <p className="small font-italic">
+            With the help of some truly incredible pro bono partners, FreeFrom was able to complete research on policies impacting survivors’ financial security in every state and D.C. One of our priorities is to ensure that all of the information in our tool is correct and up-to-date. To that end, we welcome any correction on inaccurate or incomplete information. Thank you in advance for helping us make the Policy Map and Scorecard the best tool it can possibly be!
+        </p>
+        <form>
+            <Input name="what-is-outdated" required={ true } label="What information is missing or outdated? *
```
Comment (DanielKim1) [inline components/common/ReportMissingInfo.js]: What does this `*` do? Remove the new line character?

File: components/common/ShareButtons.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFacebook, faTwitter } from '@fortawesome/free-brands-svg-icons';
+
+const ShareButtons = ({ className }) => (
+    <div className={ className }>
+        <button type="button" className="btn btn-sm btn-primary mr-2">
+            <FontAwesomeIcon icon={ faFacebook } className="mr-1" /> Share
+        </button>
+        <button type="button" className="btn btn-sm btn-primary" style={{backgroundColor: '#1b95e0', borderColor: '#1b95e0'}}>
+            <FontAwesomeIcon icon={ faTwitter } className="mr-1" /> Tweet
```
Comment (DanielKim1) [inline components/common/ShareButtons.js:12]: These are icon fonts, right? Will using these components cause accessibility issues? If so, we should make note of that

Comment (DanielKim1) [reply components/common/ShareButtons.js:12]: Checked with VoiceOver, no issues 👍

File: components/common/ReportMissingInfo.js (hunk 1 above)
Comment (DanielKim1) [reply components/common/ReportMissingInfo.js]: Oh, it's a literal `*` character. We might want to remove it, VoiceOver reads this as "What information is missing or outdated? Star. Please provide as much information as you can. Star"
```suggestion
            <Input name="what-is-outdated" required={ true } label="What information is missing or outdated?
```

**Review by DanielKim1 — APPROVED** [review]
Looks great!! :100: I have a tiny nitpick, but it's not a merge blocker.

If anyone else wants to check out this PR, apparently you can check out branches from forked repos with `gh pr checkout 89`: https://github.community/t/checkout-a-branch-from-a-fork/276/2

File: components/common/ReportMissingInfo.js (hunk 1 above)
Comment (landonreed) [reply components/common/ReportMissingInfo.js]: Ah, gotcha. Yea, I just copy pasta'ed this from what I found in `index.html`

https://github.com/RagtagOpen/freefrom-map-frontend/blob/c76018945051f37e7a8ff7266e0f51c6c0d5a88c/pages/index.js#L19-L20

But yea, I can definitely remove!

File: components/common/ShareButtons.js (hunk 2 above)
Comment (landonreed) [reply components/common/ShareButtons.js:12]: Cool, yea, this is another case where I copy pasted from `ModalButton.js`, so glad to see that it works OK!

https://github.com/RagtagOpen/freefrom-map-frontend/blob/c76018945051f37e7a8ff7266e0f51c6c0d5a88c/components/modal/ModalButton.js#L10-L12

File: components/common/ReportMissingInfo.js (hunk 1 above)
Comment (landonreed) [reply components/common/ReportMissingInfo.js]: Removed in df12ae1081070a1769c304094638da87cbf5951f

**Comment by sarahgreen** [comment]
LGTM too! I'll write up tickets for fleshing out the various parts of this page (state outline image, state scorecard, "Understanding this report" links, "Take action" links).

**Comment by sarahgreen** [comment]
Actually, one small improvement would be to left-align the two orange links. This could also be done later though.

**Pull request merged**